src/extensions/idle.rs

```diff
@@ -35,14 +35,17 @@ trait OnDrop<'a> {
 #[derive(Debug)]
 pub struct Handle<'a, T: Read + Write + 'a> {
     session: &'a mut Session<T>,
-    unsolicited_responses_tx: mpsc::Sender<UnsolicitedResponse>,
+    unsolicited_responses_tx: UnsolicitedResponseSender,
     start_time: Instant,
     keepalive: Duration,
     done: bool,
 }
 
 
 /// 'IdleIterator' allows a client to iterate over unsolicited responses during an IDLE operation.
+/// Only the unsolicited responses requested by the [`Session::request`] method will be
```

Comment:
I think you need to name the full method here?

Reply:
Oops, indeed.